src/main/resources/db/changelog/changes/05-create-users-roles-table.yaml

```diff
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-roles-table
+      author: dmytro
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    deleteCascade: true
+                    foreignKeyName: user_foreignKey

```

Comment:
Try to name you constraints so it is globally unique. Here you can see a pattern for foreign keys: https://repository.prace-ri.eu/git/help/development/database/constraint_naming_convention.md